src/test/groovy/org/gradle/android/RoomSchemaLocationWorkaroundTest.groovy

```diff
@@ -238,6 +239,77 @@ class RoomSchemaLocationWorkaroundTest extends AbstractTest {
         kotlinVersion << [ "1.3.61", "1.3.50" ].collect { VersionNumber.parse(it) }
     }
 
+    @Unroll
+    def "schemas are correctly generated when only one variant is built incrementally (Android #androidVersion)"() {
+        SimpleAndroidApp.builder(temporaryFolder.root, cacheDir)
+            .withAndroidVersion(androidVersion)
+            .withKotlinVersion(VersionNumber.parse("1.5.0"))
```

Comment:
we should be consistent with the other tests here and test all kotlin versions or use a variable here or use something like `Versions.latestForKotlin()`

Reply:
Good point.  I changed it to just use latest version - I don't think this test really has any particular sensitivity to kotlin, so I think it's fine to just run it against the one version.